rio-code/src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -273,10 +273,22 @@ public void periodic() {
         
     }
 
-    public boolean closeToCoralStation() {
+    public boolean isCoseToReef() {
```

Comment:
Typo: `isCloseToReef` and I'd prefer we consolidate between "close" and "near" and use one or the other, not both. I prefer "Near" as it's less ambiguous.